src/strings.cpp

```diff
@@ -1405,7 +1405,7 @@ static void FormatString(StringBuilder &builder, const char *str_arg, StringPara
 			case SCC_DEPOT_NAME: { // {DEPOT}
 				VehicleType vt = args.GetNextParameter<VehicleType>();
 				if (vt == VEH_AIRCRAFT) {
-					StringParameters tmp_params = StringParameters(args, 1);
+					auto tmp_params = MakeParameters(args.GetNextParameter<StationID>());
```

Comment:
`StationID` and not `DepotID`?

Reply:
VEH_AIRCRAFT means airports/airport hangars, which are referenced by StationID, see the definition of STR_FORMAT_DEPOT_NAME_AIRCRAFT.

Reply:
Ah, right, thanks.